src/edges.jl

```diff
@@ -0,0 +1,38 @@
+# Create recipe: 
+
+@Makie.recipe(Edges, grid) do scene
+    Makie.Attributes(;
+
+      # Use default theme:
+      Makie.default_theme(scene, Makie.Lines)...,
+
+      # Custom arguments:
+      color      = :brown4,
+      colormap   = :bluesreds,
+      colorrange = nothing
+    )
+end
+  
+  # Plot! especialization:
+  
+function Makie.plot!(plot::Edges{<:Tuple{Grid}})
+  
+  # Retrieve plot arguments:
+  grid      = plot[:grid][]
```

Comment:
de-referencing the observable with [] will possibly lead to problems in reactive animations.